test/integration/parachain/staging/sequential/vaults.test.ts

```diff
@@ -36,45 +35,43 @@ import {
     ESPLORA_BASE_PATH 
 } from "../../../../config";
 import { BitcoinCoreClient, newAccountId, WrappedCurrency, newVaultId, currencyIdToLiteral, CollateralIdLiteral } from "../../../../../src";
-import { encodeVaultId, getCorrespondingCollateralCurrency, issueSingle, newMonetaryAmount } from "../../../../../src/utils";
-import { callWithExchangeRate, vaultStatusToLabel } from "../../../../utils/helpers";
+import { encodeVaultId, getCorrespondingCollateralCurrencies, issueSingle, newMonetaryAmount } from "../../../../../src/utils";
+import { vaultStatusToLabel } from "../../../../utils/helpers";
 import sinon from "sinon";
 
 describe("vaultsAPI", () => {
     let oracleAccount: KeyringPair;
     let vault_to_liquidate: KeyringPair;
     let vault_to_ban: KeyringPair;
     let vault_1: KeyringPair;
-    let vault_1_id: InterbtcPrimitivesVaultId;
+    let vault_1_ids: Array<InterbtcPrimitivesVaultId>;
     let vault_2: KeyringPair;
-    let vault_2_id: InterbtcPrimitivesVaultId;
     let vault_3: KeyringPair;
-    let vault_3_id: InterbtcPrimitivesVaultId;
+    let vault_3_ids: Array<InterbtcPrimitivesVaultId>;
```

Comment:
Super minor nit, these should all be camelCase

Reply:
@savudani8 and I were tinkering with that. Turns out, camelCase gets tricky-to-read fast with those numbers thrown in. :)
At least with some of the longer names that we decided to keep snake_case in other tests. So I kept it as is for now anyway.

It feels like there is another angle to refactor here anyway, rather than having a list of `vault_<number>_<id | ids | other_stuff>` variables.